inference-engine/tests_deprecated/functional/vpu/common/layers/myriad_layers_lstm_cell.hpp

```diff
@@ -153,9 +153,204 @@ std::string tensorIteratorModel = R"V0G0N(
 </net>
 )V0G0N";
 
+std::string tensorIteratorModel_3 = R"V0G0N(
```

Comment:
I'm not sure it's allowed to add new tests with hardcoded model, even in deprecated infra. Are there any other similar tests. My vision is if test needs to have hardcoded model it should be introduced in closed tests repo.

Reply:
it is mostly artificial IR. And there is no similar other tests. If you have doubts, we can remove this test for now.

Reply:
But I guess it is ok.